redash/models/parameterized_query.py

```diff
@@ -17,16 +16,15 @@ def _pluck_name_and_value(default_column, row):
 
 
 def _load_result(query_id):
-    from redash.authentication.org_resolving import current_org
     from redash import models
 
-    query = models.Query.get_by_id_and_org(query_id, current_org)
+    query = models.Query.get_by_id(query_id)
```

Comment:
Instead of removing the `org_id` argument or relaying on `current_org`, I would pass it to `_load_result`.

Reply:
I think it should be ok to use the `org_id` value from the Query object we're rendering.

Reply:
> Instead of removing the org_id argument or relaying on current_org, I would pass it to _load_result.

Figured we'd end up with that but thought I'd see if it's avoidable first.

Reply:
Some `ParameterizedQuery` instances do not represent a persisted `Query` so we can't rely on having it all the time. Instead I just pass in the actual org 😖